src/commands/addServerNamespaceToWorkspace.ts

```diff
@@ -146,7 +146,7 @@ export async function addServerNamespaceToWorkspace(resource?: vscode.Uri): Prom
   const params = new URLSearchParams(uri.query);
   const project = params.get("project");
   const csp = params.has("csp");
-  const name = `${project ? `${project} - ` : ""}${serverName}:${namespace}${csp ? " web files" : ""}${
+  const name = `${project ? `${project} - ` : ""}${serverName}:${namespace}${!csp ? "" : ["", "/"].includes(uri.path) ? " web files" : ` web files (${uri.path})`}${
```

Comment:
Can we make the names shorter? How about `server:NS (Web)` and `server (app/path)`? We should change the project one to `server:NS (project)` as well to match.

Reply:
Like this?

![image](https://github.com/user-attachments/assets/4733f516-4230-434e-b185-01c0aaaa66ae)

Seems OK as long as I don't have a project named "Web"

Reply:
I like that better, but I think we can remove the namespace when we're tied to a specific web app. IMO that extra info isn't worth the space in that case. I see what you mean with the potential web and project conflict. Do you want to leave the "web files" suffix then? I doubt that conflict will come up in reality but it could be confusing for a user if it does.

Reply:
How about this?

![image](https://github.com/user-attachments/assets/2f53b5f4-32b3-4b7e-bd72-30e2623e7945)

Reply:
Looks good to me!

Reply:
I'm having second thoughts about the change we've made to how a project folder gets named. Some people might not like how the project name is demoted. Here's the old way:

![image](https://github.com/user-attachments/assets/c54b6508-4423-4026-85f5-379facd3565a)

Reply:
Oh, I forgot that the project used to come first! I agree, let's put that back.